Actions/ModeSet.xm

```diff
@@ -0,0 +1,62 @@
+#import <libpowercuts/libpowercuts.h>
+#import <ModeSet.h>
+
+id getOutputDevice(){
+	id routingController = [[%c(SBMediaController) sharedInstance] valueForKey:@"_routingController"];
+	id outputDevice = [[[routingController pickedRoute] logicalLeaderOutputDevice] valueForKey:@"_avOutputDevice"];
+	return outputDevice;
+}
+
+NSDictionary *modes = @{
+  @"Noise cancelling": @"AVOutputDeviceBluetoothListeningModeActiveNoiseCancellation",
+  @"Hear through": @"AVOutputDeviceBluetoothListeningModeAudioTransparency",
+  @"Inactive": @"AVOutputDeviceBluetoothListeningModeNormal"
```

Comment:
It might look better if these are the names that Apple also uses.
```suggestion
  @"Noise cancelling": @"AVOutputDeviceBluetoothListeningModeActiveNoiseCancellation",
  @"Transparency mode": @"AVOutputDeviceBluetoothListeningModeAudioTransparency",
  @"Normal": @"AVOutputDeviceBluetoothListeningModeNormal"
```